Routes/userRoutes.js

```diff
@@ -106,69 +106,75 @@ router.get("/search",verifyJWT, async (req, res) => {
     }
 
     // Return the found movies
-    res.status(200).json(movies);
+    res.status(200).json(movies.splice(0, number));
```

Comment:
when the number of movies is less than this number it would return some errors.so you can make it like this

number > movies.length? movies.length: number

Reply:
i think the slice method will manage it gracefully

Reply:
Oh, i didn't know splice will provide the service itself. Thank you for the graceful comment :D